### PR TITLE
Support find-a-user button in Preact

### PR DESCRIPTION
**Pull request by mia-pi-git**
Draft as I'm not sure the `PromptPanel` design is what I want to go with yet. Code feedback is appreciated though.

### CODE REVIEW

File: src/panels.tsx [hunk 1]
```diff
@@ -321,6 +321,16 @@ class PSMain extends preact.Component {
...
+		case 'finduser':
+			PS.addRoom({
+				id: 'finduser' as RoomID,
+				type: 'prompt',
+				message: 'Find a user',
+				parentElem: elem,
+				buttonName: "Open",
+			});
+			PS.update();
+			return true;
```
Comment (Zarel) [inline src/panels.tsx:333]: Seems like this should be a link to the `finduser` room, rather than needing to manually dispatch it?

Comment (Zarel) [reply src/panels.tsx:333]: Like just use `joinRoom` rather than `finduser`.

Comment (mia-pi-git) [reply src/panels.tsx:333]: OK.

Comment (mia-pi-git) [reply src/panels.tsx:333]: On second consideration, would that work? It would remove access to being able to set the message / buttonName params, no?

**Comment by monsanto** [comment]
Seems unlikely to happen.

**Pull request closed without merging**